packages/font/src/index.scss

```
@@ -1,4 +1,4 @@
-@use "~@fontsource/fira-sans/400.css";
-@use "~@fontsource/fira-sans/400-italic.css";
-@use "~@fontsource/fira-sans/700.css";
-@use "~@fontsource/fira-sans/700-italic.css";
+@use "pkg:@fontsource/fira-sans/400.css" as FiraSansRegular;
+@use "pkg:@fontsource/fira-sans/400-italic.css" as FiraSansItalic;
+@use "pkg:@fontsource/fira-sans/700.css" as FiraSansBold;
+@use "pkg:@fontsource/fira-sans/700-italic.css" as FiraSansBoldItalic;
```

```suggestion
@use "@fontsource/fira-sans/400.css" as FiraSansRegular;
@use "@fontsource/fira-sans/400-italic.css" as FiraSansItalic;
@use "@fontsource/fira-sans/700.css" as FiraSansBold;
@use "@fontsource/fira-sans/700-italic.css" as FiraSansBoldItalic;
```

Package Importer is niet globally supported. Als het alleen om builden gaat zou dit niet zo uitmaken, maar je staat mensen toe om deze file ook te importeren, dus dan maakt het wel uit. Je kan gebruik maken `--load-path=node_modules` in je `build:css`, zie andere comment. Dan werkt bovenstaande wijziging 👍